package-lock.json

```diff
@@ -785,20 +1015,26 @@
     },
     "node_modules/json-stable-stringify-without-jsonify": {
       "version": "1.0.1",
-      "dev": true,
-      "license": "MIT"
+      "resolved": "https://registry.npmjs.org/json-stable-stringify-without-jsonify/-/json-stable-stringify-without-jsonify-1.0.1.tgz",
+      "integrity": "sha512-Bdboy+l7tA3OGW6FjyFHWkP5LuByj1Tk33Ljyq0axyzdk9//JSi2u3fP1QSmd1KNwq6VOKYGlAu87CisVir6Pw==",
+      "dev": true
     },
     "node_modules/jsonpath-plus": {
-      "version": "8.1.0",
-      "resolved": "https://registry.npmjs.org/jsonpath-plus/-/jsonpath-plus-8.1.0.tgz",
-      "integrity": "sha512-qVTiuKztFGw0dGhYi3WNqvddx3/SHtyDT0xJaeyz4uP0d1tkpG+0y5uYQ4OcIo1TLAz3PE/qDOW9F0uDt3+CTw==",
+      "version": "10.0.0",
```

Comment:
Could you please update again? 
Your source and https://nvd.nist.gov/vuln/detail/CVE-2024-21534 report that versions `< 10.0.7` are affected. Thanks a lot 👍

Reply:
Wouldn't the semver `^10.0.0` pull in the newest `10.1.0` version? https://classic.yarnpkg.com/lang/en/docs/dependency-versions/#toc-caret-ranges

Reply:
To make the library build consistent and working with version `10.1.0`, `package-lock.json` should also be updated (again) to show explicitly that it's using a version `>= 10.0.7`.